packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/KycRequired/template.tsx

```diff
@@ -0,0 +1,160 @@
+          <FormattedMessage
+            id='modals.simplebuy.kycrequired'
+            defaultMessage='We’ve made it just as easy to buy and sell Crypto straight from your Wallet. Every Buy & Sell happens in seconds or less.'
+          />
+        </Text>
+      </FlyoutWrapper>
+
+      <SubcontentContainer>
+        <Subcontent color='grey600' weight={500}>
+          <RowNumber>
+            <NumberWrapper>
+              <NumberContainer>1</NumberContainer>
+            </NumberWrapper>
+            <NumberDescription>
+              <DisplayTitle>
+                <FormattedMessage
+                  id='modals.confirm.title.verify_identity'
+                  defaultMessage='Verify Your Identity'
+                />
+              </DisplayTitle>
+              <SubTitle>
+                <FormattedMessage
+                  id='modals.simplebuy.kycrequired.verify_identity_description'
+                  defaultMessage='To prevent identity theft or fraud, we’ll need a make sure it’s really you by uploading an ID.'
+                />
+              </SubTitle>
+            </NumberDescription>
+          </RowNumber>
+          <RowNumber>
+            <NumberWrapper>
+              <NumberContainer>2</NumberContainer>
+            </NumberWrapper>
+            <NumberDescription>
+              <DisplayTitle>
+                <FormattedMessage
+                  id='buttons.buy_crypto'
+                  defaultMessage='Buy Crypto'
+                />
+              </DisplayTitle>
+              <SubTitle>
+                <FormattedMessage
+                  id='modals.simplebuy.kycrequired.buy_crypto_description'
+                  defaultMessage='Use your Bank, Debit or Credit card to fund any crypto purchase.'
+                />
+              </SubTitle>
+            </NumberDescription>
+          </RowNumber>
+          <RowNumber>
+            <NumberWrapper>
+              <NumberContainer>3</NumberContainer>
+            </NumberWrapper>
+            <NumberDescription>
+              <DisplayTitle>
+                <FormattedMessage
+                  id='modals.simplebuy.kycrequired.swap_at_anytime'
+                  defaultMessage='Swap at Anytime'
+                />
+              </DisplayTitle>
+              <SubTitle>
+                <FormattedMessage
+                  id='modals.simplebuy.kycrequired.swap_at_anytime_description'
+                  defaultMessage='Instantly exchange your crypto for another without leaving your wallet.'
+                />
+              </SubTitle>
+            </NumberDescription>
+          </RowNumber>
+        </Subcontent>
+
+        <Button
+          fullwidth
+          size='16px'
+          height='48px'
+          nature='primary'
+          data-e2e='handleVerified'
+          onClick={() =>
+            props.identityVerificationActions.verifyIdentity(
+              2,
+              false,
+              'SBEnterAmountCheckout'
```

Comment:
Can you change this to `KycRequiredStep`

Reply:
@plondon yeah 👍